docs/podman-run.1.md

```diff
@@ -760,7 +760,7 @@ Without this argument the command will be run as root in the container.
 
 Set the user namespace mode for the container. The use of userns is disabled by default.
 
-- `host`: run in the user namespace of the caller. This is the default if no user namespace options are set. The processes running in the container will have the same privileges on the host as any other process launched by the calling user.
+- `host`: run in the user namespace of the caller and enable all privileged options (e.g., `pid=host` or `--privileged`). This is the default if no user namespace options are set. The processes running in the container will have the same privileges on the host as any other process launched by the calling user.
```

Comment:
Wait, do we actually do this? I don't know if we force --privileged on --userns=host?

Reply:
I wasn't sure, but that was in one of the two chunks of text for userns.  So I globbed it in.  Happy to remove it if it's not on point any more.

Reply:
@giuseppe Is this a rootless thing?

Reply:
It does not enable --privileged?  It just does not enable userns.

Reply:
So is it good to just remove this new wording?  "and enable ... --privileged`)"

Reply:
Yes

Reply:
removed that chunk then, pushing shortly.